### ISSUE TITLE
Ack message after long ingest

### ISSUE THREAD

**Issue by straleyb** [osulp/Scholars-Archive#1972]
### Descriptive summary

For some reason, there are certain works that take a LONG time to upload. One that I looked at and watched upload was https://ir.library.oregonstate.edu/concern/datasets/1r66j709p?locale=en which took approximately 12 minutes to upload.

Ill need to upload this work on staging (or production) and find out why it took so long.

This is causing the ACK message after upload, however, everything looks fine when you refresh the page. So its an initial error that happens ONLY after long ingest.

**Comment by simholt** [comment]
QA pass as per #1970

**Comment by simholt** [comment]
Need update on this -- AFAIK, we are suppressing all GeoNames to keep them from slowing saves down.

**Comment by luisgreg99** [comment]
I tried to recreate the Ack message locally and on staging, and only found issues in the `development` environment. 

Staging seems to be working as expected. On staging, it's now taking a few seconds to submit using the same files and metadata values in [1r66j709p](https://ir.library.oregonstate.edu/concern/datasets/1r66j709p) (successful submission).

Issues found on development environment and findings:

- Error `RSolr::Error::Http URI` during submission. 
  - From several tests, it seems like this is triggered on works with new line endings `\r\n` in textarea elements (currently being used in nested attributes fields like the abstract field). 
  - I used the same raw data (unformatted) from 1r66j709p and found that for some reason `\r\n` seems to break nested fields. 
  - This might be a solr schema issue, but it olnly affects `development` environment only.
- Error `RSolr::Error::Http URI` during `AttachFilesToWorkJob` that occurs when trying to attach multiple text files. This can be resolved by changing the `queue_adapter` to `:sidekiq`.
- Error `Expected subject (found: "GETSTMTS: access path: (includeInferred=false, s=http://opaquenamespace.org/ns/osuAcademicUnits/CJLtUOZX, p=null, o=null, c=[])")`. It turns out that this error happens because of an invalid URL value in SCHOLARSARCHIVE_TRIPLESTORE_ADAPTER_URL.
